src/ProtocolBuffers.jl

```diff
@@ -105,4 +105,10 @@ end
 export protojl, encode, ProtoEncoder, decode, decode!, ProtoDecoder
 export reserved_fields, extendable_field_numbers, oneof_field_types, field_numbers, default_values
 
+if Base.VERSION >= v"1.4.2"
```

Comment:
I'd expect our min version when we release to be Julia 1.6, since anything pre-1.6 is no longer officially supported.

Reply:
I have trouble finding the original reference, but I think that there were some issues with precompilation before 1.4.2, this version check is more about avoiding these problems than about compatibility. That being said, you have a good point notice that currently we have julia 1.7 in the Project.toml:
```
[compat]
julia = "1.7"
```
Should that be 1.6?